### DISCUSSION TITLE
Syncing successfully but logs say differently

### DISCUSSION THREAD

**Discussion by briandye** [mattwebbio/orbital-sync#97] [Q&A]
I have been syncing multiple elements, including adlists, static dhcp addresses, dns cnames, etc between two hosts successfully.  However the logs are showing failure each time... <Error: Failed to upload backup to `"http://192.168.1.99/admin". Failure: 0/1 hosts synced.>

I turn on verbose mode and all items show entries? {"host":"http://192.168.1.99","path":"/admin","status":200,"responseBody":"Processed adlist (2 entries)<br>\nProcessed adlist group assignments (2 entries)...

Any thoughts on why logs are showing failure?

**Comment by mattwebbio** [comment]
Hi @briandye! 👋 

Would you mind sharing the full verbose mode log output? I'd like to see how the server's response ends
